mce-io.c

```diff
@@ -450,23 +458,10 @@ static gboolean io_error_cb(GIOChannel *source,
 			"called with iomon == NULL!");
 		goto EXIT;
 	}
-
-	switch (iomon->error_policy) {
-	case MCE_IO_ERROR_POLICY_EXIT:
+	
```

Comment:
whitespace - maybe run indent -linux with the right options on this to avoid this?